app/src/services/reportFile.service.js

```diff
@@ -216,26 +218,29 @@ class FileService {
           questionName: response.name,
           child: null
         };
-        // check if the answer is an image
+        // check if the answer is a file
         if (response.value && response.value.startsWith("https://s3.amazonaws.com")) {
           if (payload.length < 20) {
-            // download the image
-            const image = await axios({
+            // download the file
+            const file = await axios({
               url: response.value,
               responseType: "stream",
               responseEncoding: "utf-8"
             });
+            // get the file extension
+            const splitString = response.value.split(".");
+            const [extension] = splitString.slice(-1);
             // save it to the directory - directory name should be name of report/name of question
-            const imagePath = `${record.attributes.reportName}/${response.name}/attachment.jpeg`;
-            archive.append(image.data, { name: imagePath });
-            answer.value = "image/jpeg";
+            const filePath = `${record.attributes.reportName}/${response.name}/attachment.${extension}`;
+            archive.append(file.data, { name: filePath });
+            answer.value = extension === "jpeg" ? "image/jpeg" : `audio/${extension}`;
```

Comment:
Does this need to be on the answer object instead of a variable/constant? Is the answer object or the value in the object being returned or used elsewhere?